data/src/history/manager.rs

```diff
@@ -491,6 +498,19 @@ impl Data {
 
                     true
                 }
+                message::Source::Internal(message::source::Internal::Status(status)) => {
+                    if let Some(server_message) = buffer_config.internal_messages.get(&status) {
```

Comment:
You can rename `server_message` here to `internal_message`.

Reply:
Done!